pom.xml

```diff
@@ -356,9 +355,36 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo  #517 Removing the org\takes\http\Options.java.
+                                As this class need redesign and refactoring.
+                                See the class description for the exact problems.
+                                -->
+                                <exclude>checkstyle:/src/main/java/org/takes/http/Options.java</exclude>
+
+                                <!--
+                                @todo #517 Found non-transient, non-static member.
```

Comment:
How do I add time to the pdd?
http://www.yegor256.com/2009/03/04/pdd.html I did not find the answer.